src/bootstrap/src/core/build_steps/tool.rs

```diff
@@ -744,7 +748,26 @@ impl Step for LldWrapper {
             extra_features: Vec::new(),
             allow_features: "",
             cargo_args: Vec::new(),
-        })
+        });
+
+        let libdir = builder.sysroot_target_libdir(
+            Compiler { stage: builder.top_stage, host: self.target },
+            self.target,
+        );
+        let libdir_bin = libdir.parent().unwrap().join("bin");
```

Comment:
Question: can this use `sysroot_target_bindir`?

Reply:
Yeap, I forgot that actually exist.